File: output/model-compare-agentic/20260308-232041/workspaces/agentic_python_metrics_contract/distilled_q4_k_m/repeat-1/runtime/metrics.py
class MetricsTracker:
    def __init__(self) -> None:
        self.turns_total = 0
        self.invalid_output_count = 0
        self.action_applied_count = 0
        self.action_rejected_count = 0
        self.queued_actions = 0

    def snapshot(self) -> dict:
        return {
            "turns_total": self.turns_total,
            "invalid_output_count": self.invalid_output_count,
            "action_applied_count": self.action_applied_count,
            "action_rejected_count": self.action_rejected_count,
            "queued_actions": self.queued_actions,
        }

File: output/qwen35-27b-vs-qwen36-27b-q5xl/20260423-095434/workspaces/agentic_ts_metrics_contract_patch/qwen36_27b_ud_q5_xl/repeat-2/apps/game-runtime/src/runtime/metrics.ts
export type RuntimeMetrics = {
  turnsTotal: number;
  invalidOutputCount: number;
  actionAppliedCount: number;
  actionRejectedCount: number;
  queuedActions: number;
};

export class MetricsTracker {
  private metrics: RuntimeMetrics = {
    turnsTotal: 0,
    invalidOutputCount: 0,
    actionAppliedCount: 0,
    actionRejectedCount: 0,
    queuedActions: 0,
  };

  snapshot(): RuntimeMetrics {
    return { ...this.metrics };
  }
}
